src/handlers/dns/DnsStreamHandler.cpp

```diff
@@ -309,6 +313,30 @@ bool DnsStreamHandler::_filtering(DnsLayer &payload, [[maybe_unused]] PacketDire
     return true;
 }
 
+const std::bitset<64> DnsStreamHandler::_process_dns_groups(const Configurable *metrics_config)
```

Comment:
I could not return a pointer here because I could initialize the group into a DnsStreamHandler variable before passing it to StreamMetricsHandler.